courseScheduleII.js

```diff
@@ -0,0 +1,113 @@
+/*
+For this approach, we can represent the input as a dependency graph where in order to take a course, 
```

Comment:
This is not phrased very well. The statement "in order to take a course, you have to complete a prereq" implies that each course has 1 prereq, which is not true. I'd say something like this: "For this approach, we can represent our input as a dependency graph. For each course, all of the incoming edges represent prereqs (i.e. courses we must complete BEFORE taking our current course)."

Try rewriting something like that in your own words.

Reply:
For this approach, we can represent the input as a dependency graph where each course is followed by one or more prereqs and in order to take to take a course, you have to complete the required courses before the current one. This is the same as node having multiple incoming edges.

Reply:
Looks good.